static/des_components/des-y3a2/gold/badregions.html

```diff
@@ -0,0 +1,234 @@
+<dom-module id="des-y3a2-gold-badregions">
```

Comment:
@aureliocarnero There appear to be four duplicates of this file with different names.

Reply:
This is ongoing work and actually, they will dissapear later. For now, we can just leave it